scripts/steps/build-api.js

```diff
@@ -110,6 +104,49 @@ const UNWRAP_UTILITIES = new Set([
   'Readonly'
 ])
 
+const EXCLUDED_SUBMENU_KINDS = new Set(['Type alias', 'Interface', 'Namespace'])
+const EXCLUDED_TREE_KINDS = new Set(['Namespace'])
+
+const GROUPS = {
+  Actions: 'Actions',
+  Client: 'Client',
+  Core: 'Core',
+  Other: 'Other',
+  React: 'React',
+  Server: 'Server',
+  State: 'State',
+  Tests: 'Tests',
+  Types: 'Types',
+  Vue: 'Vue'
+}
+
+// More specific group conditions should be placed higher
+const GROUPS_CONDITION = {
+  [GROUPS.Types]: node =>
+    node.kindString === 'Type alias' || node.kindString === 'Interface',
+  [GROUPS.React]: node => isSource(node, 'react'),
+  [GROUPS.Vue]: node => isSource(node, 'vue'),
+  [GROUPS.Tests]: node => isSource(node, 'test'),
+  [GROUPS.Actions]: node => isSource(node, 'logux-actions'),
+  [GROUPS.Client]: node => isSource(node, 'logux-client'),
+  [GROUPS.Core]: node => isSource(node, 'logux-core'),
```

Comment:
Nope. Moved the whole project to the group is not care about user.

For instance, `logux-core` contains many small internal types, which will interrupt from learning what classes are important in the core.

You need to walk through TypeDoc classes & types and manually put each class to the right group.

Only important and useful to learning Logux Core classes should be in `Core` group. All small and non-important types and classes should be in `Other`.

The same applies to all other groups.